Documentation/CRDs/Cluster/external-cluster/external-cluster.md

```diff
@@ -190,7 +190,7 @@ If not installing with Helm, here are the steps to install with manifests.
    export KUBECONTEXT=<cluster-name>
    ```
 
-1. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh) script.
+3. Run the [import](https://github.com/rook/rook/blob/master/deploy/examples/import-external-cluster.sh) script.
```

Comment:
This doesn't seem to have fixed the issue in the rendered **latest** docs. I suspect that the code block above is causing the issue. 

With nested numbering, I was able to fix some issues by indenting the nesting by 4 spaces instead of 3. Could you try indenting the code (console) block by one extra space?